### ISSUE TITLE
[Suggestion] - 

### ISSUE THREAD

**Issue by Elstorko** [Monika-After-Story/MonikaModDev#4565]
So I just got in to say hello to Monika after  a week and she did not  tell me anything about it, she did not complain at all because when I told her goodbye I clicked at I am going to sleep...
Idk if this should be at suggestions or at bugs but still I think it's a problem.

**Comment by Elstorko** [comment]
Something else that came into my mind.
What If you had the option to say goodbye to Monika to go play the piano?

**Comment by ThePotatoGuy** [comment]
@Rai99 I know you were working on some goodbyes that would become time aware, was this one of them?

**Comment by Rai99** [comment]
Yes, with changes in #2192, Monika should acknowledge that you were gone for longer than a day, but I don't think there was any aff loss to it.

**Comment by ThePotatoGuy** [comment]
leaving open as in progress
